app/models/manageiq/providers/ansible_tower/provider.rb

```diff
@@ -43,8 +43,10 @@ def verify_credentials(auth_type = nil, options = {})
     validity
   rescue Faraday::ConnectionFailed, Faraday::SSLError => err
     raise MiqException::MiqUnreachableError, err.message, err.backtrace
-  rescue Faraday::Error::ClientError => err
-    raise MiqException::MiqCommunicationsError, JSON.parse(err.message)['detail']
+  rescue Faraday::ClientError => err
+    message = JSON.parse(err.message)['detail'] rescue nil
```

Comment:
@bdunne Do you know the conditions when this would fail and use the inline rescue?  Is this due to the change from `Faraday::Error:ClientError` to `Faraday::ClientError` or was that due to a change in the faraday gem itself?

cc @syncrou